[2025-02-08T01:47:56.098+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-08T01:47:56.215+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Incident_Report_Generation.procesar_logs_restart manual__2025-02-08T01:44:13.666113+00:00 [queued]>
[2025-02-08T01:47:56.301+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Incident_Report_Generation.procesar_logs_restart manual__2025-02-08T01:44:13.666113+00:00 [queued]>
[2025-02-08T01:47:56.304+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-02-08T01:47:56.387+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): procesar_logs_restart> on 2025-02-08 01:44:13.666113+00:00
[2025-02-08T01:47:56.427+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=127) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-08T01:47:56.429+0000] {standard_task_runner.py:72} INFO - Started process 147 to run task
[2025-02-08T01:47:56.423+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Incident_Report_Generation', 'procesar_logs_restart', 'manual__2025-02-08T01:44:13.666113+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/Incident_Report_Generation.py', '--cfg-path', '/tmp/tmp9g4wkb3y']
[2025-02-08T01:47:56.435+0000] {standard_task_runner.py:105} INFO - Job 37: Subtask procesar_logs_restart
[2025-02-08T01:47:56.599+0000] {task_command.py:467} INFO - Running <TaskInstance: Incident_Report_Generation.procesar_logs_restart manual__2025-02-08T01:44:13.666113+00:00 [running]> on host 0b0ecc2e9e80
[2025-02-08T01:47:56.741+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Incident_Report_Generation' AIRFLOW_CTX_TASK_ID='procesar_logs_restart' AIRFLOW_CTX_EXECUTION_DATE='2025-02-08T01:44:13.666113+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-08T01:44:13.666113+00:00'
[2025-02-08T01:47:56.743+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-08T01:47:56.744+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-08T01:47:56.745+0000] {logging_mixin.py:190} INFO - Current task name:procesar_logs_restart state:running start_date:2025-02-08 01:47:56.242523+00:00
[2025-02-08T01:47:56.746+0000] {logging_mixin.py:190} INFO - Dag name:Incident_Report_Generation and current dag run status:running
[2025-02-08T01:47:56.747+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-08T01:47:56.792+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-08T01:47:56.805+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-08T01:47:56.806+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Incident_Report_Generation, task_id=procesar_logs_restart, run_id=manual__2025-02-08T01:44:13.666113+00:00, execution_date=20250208T014413, start_date=20250208T014756, end_date=20250208T014756
[2025-02-08T01:47:56.825+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-08T01:47:56.825+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-08T01:47:56.826+0000] {logging_mixin.py:190} INFO - Dag name:Incident_Report_Generation queued_at:2025-02-08 01:44:14.058854+00:00
[2025-02-08T01:47:56.827+0000] {logging_mixin.py:190} INFO - Task hostname:0b0ecc2e9e80 operator:PythonOperator
[2025-02-08T01:47:56.859+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-08T01:47:56.883+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-08T01:47:56.890+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
